golang.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "context"
+
+//here is a line coment
+
+// FileGetter is usd to retriv de code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
```

Comment:
Typo inside comment in token 'returs'. Possible corrections:return (78),returns (63),returs (1)